ietf/static/js/list.js

```diff
@@ -62,12 +60,21 @@ $(document)
                 // get field classes from first thead row
                 var fields = $(header_row)
```

Comment:
I'm going to approve this, but this bit of new code is particularly hard to understand. I wonder if it could be written in a way that would promote better maintenance?